solution/ui/regulations/js/src/components/SupplementalContent.vue

```diff
@@ -1,4 +1,9 @@
 <template>
+  <div>
+    <a v-if="selectedPart" v-on:click="clearSection" style="color:#5B616B">View All Subpart {{subparts[0]}} Resources  ({{resourceCount}})</a>
```

Comment:
This link should be `font-size: 12px`. Also, the "View All Subpart resources" part of the button label should be `font-weight: 600`. Can wrap in a span.